app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = current_user.payments
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+    @unpaid_ticket_purchases = current_user.ticket_purchases.unpaid.by_conference(@conference)
+  end
+
+  def create
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+
+    @payment = Payment.new payment_params.merge(stripe_customer_email: params[:stripeEmail],
+                                                stripe_customer_token: params[:stripeToken],
+                                                amount: @total_amount_to_pay.cents,
```

Comment:
Since you are calculating the amount to pay inside the Payment model, why do you need to pass it here?